notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>Maybe this could help you bypass creating and storing a tmp file?</p><pre class="ql-syntax" spellcheck="false">ret = s3c.upload_df_to_s3(
&nbsp; &nbsp; preds_kpi,
&nbsp; &nbsp; s3_prefix=config.BASE_INFER_KPI_TABLE_S3_PREFIX,
&nbsp; &nbsp; s3_key="upload-filename.parquet",
&nbsp; &nbsp; filetype=S3FileType.PARQUET,
&nbsp; &nbsp; index=False,
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>great, thanks!</p>